README.md

```diff
@@ -29,7 +30,7 @@ Battery Service (BAS)| | ✓ |
 Cycling Speed and Cadence Service (CSCS) | | ✓ |
 Cycling Power Service (CPS) | | ✓ |
 Elite Sterzo Steering Service (STERZO)| | ✓ |
-FiTness Machine Service (FTMS) | | |✓
+FiTness Machine Service (FTMS) | | ✓ |
```

Comment:
I think we can actually remove the "Not supported" column altogether. I added it originally to emphasise the fact FTMS isn't supported yet.